### ISSUE TITLE
NumFOCUS funding

### ISSUE THREAD

**Issue by waynelapierre** [Rdatatable/data.table#5675]
pandas has been funded by NumFOCUS since 2015. data.table is superior to pandas in terms of syntax and speed and has been a vital part of R's data science ecosystem. Is it possible to get NumFOCUS funding for data.table?  
https://numfocus.org/project/pandas

**Comment by jangorecki** [comment]
Great idea. Question if numfocus funds R projects as well. Brief look at their website shows only python.

**Comment by kadyb** [comment]
> Question if numfocus funds R projects as well.

Rather yes. The [website](https://numfocus.org/projects-overview) states this information: 

> We support projects in various languages, including Python, Julia, R, C++, and JavaScript.

**Comment by waynelapierre** [comment]
rOpenSci has been funded by NumFOCUS since 2014.
https://numfocus.org/project/ropensci

**Comment by tdhock** [comment]
https://numfocus.org/projects-overview explains that creating a formal governance document is required before data.table could apply for fiscal sponsorship from NumFOCUS. 
```
Additional Requirements for Fiscal Sponsorship:

    A transparent, publicly visible governance model
    A roadmap outlining high priority work areas
```

**Comment by tdhock** [comment]
Now that we have a code of conduct, we can apply for NumFOCUS funding. The next deadline is April 15. Can someone please volunteer to draft the application, and share it with our community here on this issue, before submitting it to NumFOCUS for review before that deadline?

**Comment by TysonStanley** [comment]
@tdhock thanks for re-upping this. I definitely draft an application but would like to hear others' opinion on what the up to $10k could be used on (possibly some developer time for specific feature requests, travel, etc.), including the vision for where the project could go with the funding.

**Comment by waynelapierre** [comment]
As a side note, there are also funding opportunities at https://summerofcode.withgoogle.com/. The deadline is probably too close.

**Comment by tdhock** [comment]
Thanks Wayne! I actually created a web page for GSOC, https://github.com/rstats-gsoc/gsoc2024/wiki/data.table and it is definitely not too late. We need to find an interested/motivated student though!

**Comment by Anirban166** [comment]
Agreed, and @tdhock thanks for bringing this up in our meeting yesterday - I made it a point to mention trying GSoC or the summer of code programs in general for the students I'm interviewing that have interest in contributing, since they've been using data.table already or have used in the past. (For e.g., this morning I talked to @damirpolat who is looking for opportunities as he'll graduate soon and he sounded willing to go for it, and I linked that page during my Zoom meeting with him)

It'd be interesting to see what projects we could come up with for multiple months worth of work. Great potential for making progress towards major bug fixes and features!

**Comment by Anirban166** [comment]
I second that, and @tdhock thanks for bringing this up in our meeting yesterday - I made it a point to mention trying GSoC or the summer of code programs in general for the students I'm interviewing that have interest in contributing, since they've been using data.table already or have used in the past. (For e.g., this morning I talked to @damirpolat who is looking for opportunities as he'll graduate soon and he sounded willing to go for it, and I linked that page during my Zoom meeting with him)

It'd be interesting to see what projects we could come up with for multiple months' worth of work. Great potential for making progress towards major bug fixes and features!

**Comment by tdhock** [comment]
`data.table` has been accepted as a NumFOCUS fiscally sponsored project.

[data.table Sponsored Project Welcome Letter (3).pdf](https://github.com/user-attachments/files/20342189/data.table.Sponsored.Project.Welcome.Letter.3.pdf)

All community members are welcome to join the onboarding meeting is next week: Thursday May 29, 2025 ⋅ 10am – 11am (Eastern Time - Toronto)
Zoom meeting ID: 81325103382
passcode: 723945

The letter has the following action items. Can someone please volunteer to submit corresponding PRs?

* Add the “Powered by NumFOCUS” badge for your website and GitHub.
Markdown for adding NumFOCUS badge to project websites: [![Powered by
NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](http://numfocus.org)
* Add the Fiscal Sponsor Readme Attribution to your project’s GitHub:
https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md
* Add the NumFOCUS sponsored project logo ( with link to numfocus.org ) to your
project’s website (under About) and language indicating that you are a sponsored project
of NumFOCUS. For example: “[Project] is a fiscally sponsored project of NumFOCUS, a
nonprofit dedicated to supporting the open-source scientific computing community. If you
like [Project] and want to support our mission, please consider making a [donation](link
to Salsa campaign) to support our efforts.” (*We will provide you the campaign link to
include on your website once you’ve sent us the marketing materials listed above)
* Link to the donation page. Please use this exact language alongside the donation link:
“NumFOCUS is a 501(c)(3) non-profit charity in the United States; as such, donations to
NumFOCUS are tax-deductible as allowed by law. As with any donation, you should
consult with your personal tax adviser or the IRS about your particular tax situation.”

**Comment by tdhock** [comment]
There is another action item related to revising the code of conduct:

Action Item: As part of the acceptance process, NumFOCUS assumes that your project will be accepting our Code of Conduct (CoC). However, you do have the option to opt-out and use an alternative CoC. The project has 3 weeks following the start of onboarding to make the decision on opting-in or opting-out of the NumFOCUS CoC. 

        The first step is to watch this[ Introductory Video](https://www.youtube.com/watch?v=Y8u4qtD6ZwU&feature=youtu.be).You can find important information related to the CoC regardless of your project final decision. You can review the NumFOCUS CoC [HERE](https://numfocus.org/code-of-conduct).

        Please ensure you post the CoC you will use to your online platforms (GitHub, websites, etc).
    If you have any questions about the CoC decision making process, I've copied Kamila on this email who is our Advocacy Program Lead, and is heading up the work being done on this. Thanks again, have a great weekend, Kari

**Comment by jangorecki** [comment]
Thanks @kbodwin for preparing DT side of the call!
